Added Synology Z-Wave USB stick configuration option

andrew-curtis opened the pull request:
added Docker Terminal command for creating a new Synology Docker container with access to a connected Z-Wave USB stick.

Would be good to add similar config switches for the other platforms too?

**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

andrew-curtis committed:
Added Synology Z-Wave USB stick configuration 

added Docker Terminal command for creating a new Synology Docker container with access to a connected Z-Wave USB stick.

Would be good to add similar config switches for the other platforms too?

DubhAd commented on source/_docs/installation/docker.markdown:
```diff
@@ -77,6 +77,18 @@ The steps would be:
 * Click on "Next" and then "Apply"
 * Your Home Assistant within Docker should now run and will serve the web interface from port 8123 on your Docker host (this will be your Synology NAS IP address - for example `http://192.168.1.10:8123`)
 
+To use a Z-Wave USB stick for Z-Wave control, the HA Docker container needs extra configuration to access to the USB stick. While there are multiple ways to do this, the least privileged way of granting access can only be performed via the Terminal, at the time of writing. See this page for configuring Terminal acces to your Synology NAS. https://www.synology.com/en-global/knowledgebase/DSM/help/ ...
```
HA -> Home Assistant

Make the link a link

DubhAd commented on source/_docs/installation/docker.markdown:
```diff
@@ -77,6 +77,18 @@ The steps would be:
 * Your Home Assistant within Docker should now run and will serve the web interface from port 8123 on your Docker host (this will be your Synology NAS IP address - for example `http://192.168.1.10:8123`)
 
+To use a Z-Wave USB stick for Z-Wave control, the HA Docker container needs extra configuration to access to the USB stick. While there are multiple ways to do this, the least privileged way of granting access can only be performed via the Terminal, at the time of writing. See this page for configuring Terminal acces to your Synology NAS. https://www.synology.com/en-global/knowledgebase/DSM/help/ ...
+See this page for accessing the Terminal via SSH. https://www.synology.com/en-global/knowledgebase/DSM/tutorial/General_Setup/How_to_login_to_DSM_with_root_permission_via_SSH_Telnet
```
Make the link a link

andrew-curtis replied:
```suggestion
See this page for accessing the Terminal via SSH. `https://www.synology.com/en-global/knowledgebase/DSM/tutorial/General_Setup/How_to_login_to_DSM_with_root_permission_via_SSH_Telnet`
```

DubhAd commented on source/_docs/installation/docker.markdown:
````diff
@@ -77,6 +77,18 @@ The steps would be:
+```bash
+sudo docker run --restart always -d --name="homeassistant" -v /PATH_TO_YOUR_CONFIG:/config --device=/PATH_TO_YOUR_USB_STICK -e TZ=Australia/Melbourne --net=host homeassistant/home-assistant
+```
+Complete the remainder of the Z-Wave configuration by following the instructions here. https://www.home-assistant.io/docs/z-wave/installation
````
Link it:

`Complete the remainder of the Z-Wave configuration by [following the instructions](/docs/z-wave/installation)`

andrew-curtis replied:
```suggestion
Complete the remainder of the Z-Wave configuration by [following the instructions here.] (/docs/z-wave/installation)
```

DubhAd commented on source/_docs/installation/docker.markdown:
```diff
@@ -77,6 +77,18 @@ The steps would be:
+Adjust the following Terminal command as follows 
+* Replace `/PATH_TO_YOUR_CONFIG` points at the folder where you want to store your configuration
+* Replace `/PATH_TO_YOUR_USB_STICK` matches the path for your USB stick (e.g. `/dev/ttyACM0` for most Synology users)
+* Replace "Australia/Melbourne" with [your timezone](http://en.wikipedia.org/wiki/List_of_tz_database_time_zones).and run it in Terminal.  
```
Lose the full stop after the bracket and before the and

andrew-curtis replied:
```suggestion
* Replace "Australia/Melbourne" with [your timezone](http://en.wikipedia.org/wiki/List_of_tz_database_time_zones) 
Run it in Terminal.  
```

andrew-curtis commented on source/_docs/installation/docker.markdown:
```diff
@@ -77,6 +77,18 @@ The steps would be:
 * Click on "Next" and then "Apply"
 * Your Home Assistant within Docker should now run and will serve the web interface from port 8123 on your Docker host (this will be your Synology NAS IP address - for example `http://192.168.1.10:8123`)
 
+To use a Z-Wave USB stick for Z-Wave control, the HA Docker container needs extra configuration to access to the USB stick. While there are multiple ways to do this, the least privileged way of granting access can only be performed via the Terminal, at the time of writing. See this page for configuring Terminal acces to your Synology NAS. https://www.synology.com/en-global/knowledgebase/DSM/help/ ...
```
```suggestion
To use a Z-Wave USB stick for Z-Wave control, the HA Docker container needs extra configuration to access to the USB stick. While there are multiple ways to do this, the least privileged way of granting access can only be performed via the Terminal, at the time of writing. See this page for configuring Terminal acces to your Synology NAS. `https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/AdminCenter/system_terminal`
```

andrew-curtis replied:
updated (I think). Still getting familiar with GitHub and PRs etc.

andrew-curtis committed:
Apply suggestions from code review

trying my best to stumble through this. I'm still very new to this and have only read the Home Assistant doco on contributing. Hopefully i have done this right?

frenck committed:
:pencil2: Tweaks

frenck reviewed (approved):
Thanks, @andrew-curtis! 👍
